shimmingtoolbox/cli/mask.py

```diff
@@ -0,0 +1,185 @@
+        # of the cube mask
+        mask_cb = mask_cb.astype(int)
+        nii_img = nib.Nifti1Image(mask_cb, nii.affine)
+        fname_mask = os.path.join(output, 'mask.nii.gz')
+        nib.save(nii_img, fname_mask)
+        click.echo(f"The filename for the output mask is: {os.path.abspath(fname_mask)}")
+        return output
+
+    else:
+        raise ValueError("The nifti file does not have 3 dimensions.")
+
+
+@mask.command(context_settings=CONTEXT_SETTINGS,
+              help=f"Creates a square mask from the input file. Returns an output nifti file with square mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Complete input path of the nifti file to"
+                                                                              " mask")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option("-len_dim1", required=True, type=int, help="Length of the side of the square along first dimension (in "
+                                                         "pixels)")
+@click.option("-len_dim2", required=True, type=int, help="Length of the side of the square along second dimension (in "
+                                                         "pixels)")
+@click.option("-center_dim1", type=int, default=None, help="Center of the square along first dimension (in "
+                                                           "pixels). If no center is provided, the middle is "
+                                                           "used.")
+@click.option("-center_dim2", type=int, default=None, help="Center of the square along second dimension (in "
+                                                           "pixels). If no center is provided, the middle is "
+                                                           "used.")
+def square(fname_input, output, len_dim1, len_dim2, center_dim1, center_dim2):
+    """
+            Creates a square mask from the input file. Returns an output nifti file with square mask.
+
+            Args:
+                fname_input (str): Complete input path of the nifti file to mask.
+                output (str): Output folder for square mask.
+                len_dim1 (int): Length of the side of the square along first dimension (in pixels).
+                len_dim2 (int): Length of the side of the square along second dimension (in pixels).
+                center_dim1 (int): Center of the square along first dimension (in pixels). If no center is
+                                provided, the middle is used.
+                center_dim2 (int): Center of the square along second dimension (in pixels). If no center is
+                                provided, the middle is used.
+
+            Return:
+                output (str): Output nifti file with square mask.
+            """
+    # Create the folder where the nifti file will be stored
+    if not os.path.exists(fname_input):
```

Comment:
should be checked by the parser

Reply:
argparse can check for that-- i'm sure click can do it (otherwise that's a strong cons against click)

Reply:
I removed it as it was not necessary